### DISCUSSION TITLE
Shortcode or widget to show "# of visits to this post"

### DISCUSSION THREAD

**Discussion by arnelap** [ibericode/koko-analytics#233] [Ideas]
just a total counter that shows the total views of the post you're viewing, to normal website visitors (not logged in)

**Comment by dannyvankooten** [comment]
As of 893cf5ceb58664aae1d196dd3aab8f9afcf07dc0, there is a shortcode for this.

To show the total # of visitors for the current page over the last 10 years:

```
[koko_analytics_counter]
```

To show the total # of pageviews for the entire site over the last 30 days:

```
[koko_analytics_counter days="30" metric="pageviews" global="true"]
```


To show the total # of pageviews for the current page over the last 30 days:

```
[koko_analytics_counter days="30" metric="pageviews"]
```
